keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -296,21 +584,20 @@ revert to the old behavior where rolling update will proceed one pod at a time.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-We will restore the desired behavior for StatefulSets for which the maxunavailable field wasn't deleted after 
+We will restore the desired behavior for StatefulSets for which the maxunavailable field wasn't deleted after
 the feature gate was disabled.
 
 ###### Are there any tests for feature enablement/disablement?
-yes, there are unit tests which make sure the field is correctly dropped 
+yes, there are unit tests which make sure the field is correctly dropped
 on feature enable and disabled
 
 ### Rollout, Upgrade and Rollback Planning
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-A rollout or rollback of this feature can fail if there is a bug which causes the kube-apiserver or 
+A rollout or rollback of this feature can fail if there is a bug which causes the kube-apiserver or
 the kube-controller-manager to start crashing when the feature flag is enabled.
```

Comment:
That kind of case should be verified in integration testing. This question is more of what can happen assuming the feature is used (ie. field set) but we turn the feature off. What impact it may have on the workload.